README.md

```diff
@@ -176,12 +176,10 @@ module.exports = function(environment) {
 
 ```js
 // app/routes/applicaton.js
-import moment from 'moment';
-
 export default Ember.Route.extend({
+  moment: Ember.inject.service(),
   beforeModel() {
-    // sets the application locale to Spanish
-    moment.locale('es');
+    this.set('moment.locale', 'es');
```

Comment:
Where would the best place for this to live than since the majority use case would be setting at runtime since its user dependent